src/client/app/_shared/item.component.ts

```diff
@@ -156,7 +156,12 @@ export class ItemComponent implements OnInit {
                                           && this.context !== 'Coin'
                                           && this.item.isSackable,
                               execute: () => this.doColyseusMoveAction('S')  },
-
+                          
+    { label: 'To Locker',     visible: () => this.context !== 'Locker'
```

Comment:
Call it a Kubby instead. You can't use Locker as the context because L is Left Hand already.